hack/generate-proto.sh

```diff
@@ -43,6 +43,9 @@ APIMACHINERY_PKGS=(
     k8s.io/api/core/v1
 )
 
+export GO111MODULE=on
```

Comment:
Borrowed that hack from argo workflows https://github.com/argoproj/argo-workflows/blob/4e450e250168e6b4d51a126b784e90b11a0162bc/Makefile#L323